mitosheet/src/api.tsx

```diff
@@ -539,6 +541,52 @@ export default class MitoAPI {
         return stepID
     }
 
+    async editGraphDelete(
+        graphID: GraphID,
+    ): Promise<void> {
+
+        await this.send<string>({
+            'event': 'edit_event',
+            'type': 'graph_delete_edit',
+            step_id: getRandomId(),
+            'params': {
+                'graph_id': graphID
+            }
+        }, {})
+    }
+
+    async editGraphDuplicate(
+        originalGraphID: GraphID,
```

Comment:
in all other places, we use the word `old` to say it's old vs new. Can we change original -> old in all places (including throughout the step)?